test/built-ins/Set/prototype/union/converts-negative-zero.js

```diff
@@ -0,0 +1,34 @@
+// Copyright (C) 2023 Anthony Frehner. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: pending
+description: Set.prototype.union converts -0𝔽 to +0𝔽
+info: |
+    7.b.ii. If nextValue is -0𝔽, set nextValue to +0𝔽.
+features: [Set-methods]
+includes: [compareArray.js]
+---*/
+
+const s1 = new Set([1]);
+const s2 = new Set([-0]);
+let expects = [1, +0];
+let combined = s1.union(s2);
+
+assert.compareArray([...combined], expects);
```

Comment:
I was worried for a moment, but I verified that `compareArray` catches the difference (though the error message is awful: `Test262Error: Expected [0] and [0] to have the same contents.`). No action required on this comment.